src/AddItem/AddItem.js

```diff
@@ -78,10 +71,26 @@ class AddItem extends Component {
   _renderIcon = () => {
     const { size, theme } = this.props;
 
-    const image = theme === 'image';
-    const iconElement = ICONS[image ? 'custom' : size];
+    const isImageIcon = theme === 'image';
 
-    return iconElement;
+    return (
+      <ThemeProviderConsumerBackwardCompatible
+        defaultIcons={{
+          AddItemButton: {
+            tiny: () => <Add width="26" height="26" />,
+            small: AddItemSmall,
+            medium: AddItemMedium,
+            large: AddItemLarge,
+            image: () => <AddMedia width="31" height="31" />,
+          },
```

Comment:
consider refactoring it out to a static variable to avoid re-renders of the function component.

const 
```jsx
const AddItemButtonIcons = {
  tiny: () => <Add width="26" height="26" />,
  small: AddItemSmall,
  medium: AddItemMedium,
  large: AddItemLarge,
  image: () => <AddMedia width="31" height="31" />,
};
```

```suggestion
          AddItemButton: AddItemButtonIcons
```